RegExAndExceptionHandling/RegExandExceptionHandling/PhoneVerifier.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace RegExandExceptionHandling
+{
+    /// <summary>
+    /// Verifies phone numbers using RegEx
```

Comment:
Comments should be complete sentences and should end with a period (C3, C4) (-0.5)